docs/docs/en/guide/resource/configuration.md

```diff
@@ -2,13 +2,23 @@
 
 The Resource Center is usually used for operations such as uploading files, UDF functions, and task group management. You can appoint the local file directory as the upload directory for a single machine (this operation does not need to deploy Hadoop). Or you can also upload to a Hadoop or MinIO cluster, at this time, you need to have Hadoop (2.6+) or MinIO or other related environments.
 
+## Local File Resource Configuration
+For a single machine, you can choose to use local file directory as the upload directory (no need to deploy Hadoop) by making the following configuration.
+
+### Configuring the common.properties
+
+Configure the file in the following paths: `api-server/conf/common.properties` and `worker-server/conf/common.properties`.
+
+- Change 'data.basedir.path' to the local directory path. Please make sure the directory exists and have read write permissions, such as: `data.basedir.path=C:/tmp/dolphinscheduler`.
+- Modify two fields, `resource.storage.type=HDFS` and `fs.defaultFS=file:///`.
+
 ## HDFS Resource Configuration
 
 When it is necessary to use the Resource Center to create or upload relevant files, all files and resources will be stored on HDFS. Therefore the following configuration is required.
 
 ### Configuring the common.properties
 
-After version 3.0.0-alpha, if you want to upload resources using HDFS or S3 from the Resource Center, you will need to configure the following paths The following paths need to be configured: `api-server/conf/common.properties` and `worker-server/conf/common.properties`. This can be found as follows.
+After version 3.0.0-alpha, if you want to upload resources using HDFS or S3 from the Resource Center, you will need to configure the following paths: `api-server/conf/common.properties` and `worker-server/conf/common.properties`. This can be found as follows.
```

Comment:
The text repeats itself here. `you will need to configure the following paths` and `The following paths need to be configured`. I deleted one of them and kept the other.

Reply:
> I think we should keep it as it is which is more accurate.

The text repeats itself here. `you will need to configure the following paths` and `The following paths need to be configured`. I deleted one of them and kept the other.

Reply:
My mistake. Should we keep ```The following paths need to be configured``` instead of ```you will need to configure the following paths```? @GavinGYM